test/scalability/default_rangespec.yaml

```diff
@@ -22,8 +16,8 @@ clusterQueueClassesMinUsage:
   cq: 56 #%
 
 wlClassesMaxAvgTimeToAdmissionMs:
-  # Average value 6666 (+/- 14%), setting at +20%
-  large: 8_000
+  # Average value 6666 (+/- 14%), setting at +35%
```

Comment:
Can we have some methodology that allows us to estimate the expected ratio of failures? Maybe something along the lines [here](https://github.com/kubernetes-sigs/kueue/issues/2066#issuecomment-2077592505)? 

Otherwise it is unclear that this +35% mean. 

I'm ok to leave it as a follow up.

Reply:
We'll need some kind of stable way of running an collecting the results, we don't at this point, and as we just saw the CI env is not very predictable.

Reply:
Right, that is why I'm thinking about fitting the standard distribution of the results, with this approach we can account for the variance. If the variance is high, we will get a high threshold, that's ok.  Still, it will let us set the threshold to control the ratio of failures.

Reply:
Why are we discussing things like this in a PR that should be a quick fix for a bug which may or may not block the CI pipeline. (Good thing that the load on the CI clusters has decrease and the tests are now passing :) )

Reply:
I'm totally for a quick fix, I'm even surprised there is more changes than just bumping the thresholds. 

I just point out it will be good to follow up with some more methodological approach.